blog.md

```diff
@@ -0,0 +1,100 @@
+## Take a look at your repository from a new perspective with Git Machete Plugin!
+
+Git and IntelliJ-based software is currently one of the most important tools in software development.
+The version control system provided with the JetBrains's IDE products
+allows performing pretty much all everyday VCS actions directly via the IDE.
+Among the many useful benefits of the provided IDE and VCS integration,
+the full support of GitHub Pull Requests introduced in IDEA 2020.2 must be emphasized.
+
+Creating and maintaining small, focused, easy-to-review PRs is a comfortable and desired approach
+for both the authors and, especially, the reviewers.
+[VirtusLab](https://virtuslab.com/) created a plugin that eases the management of 2+ branches in the repository
+while also promoting creation of small PRs &mdash;
+**a git repository organizer, and a rebase/merge/push/pull automation tool, named Git Machete.**
+
+It allows to look at the state of the repository from a previously inexperienceable perspective.
```

Comment:
https://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english/103971